README.md

```diff
@@ -1,3 +1,35 @@
 # Starknet in Rust
 
 `starknet_in_rust` is a implentation of [Starknet](https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/starknet) in Rust.
+It makes use of [cairo-rs](https://github.com/lambdaclass/cairo-rs), the Rust implementation of the Cairo virtual machine.
+
+## Requisites
+- Rust 1.62
+- A working installation of cairo-lang (for compiling the cairo files)
```

Comment:
Do we need to ping a particular version of `cairo-lang`?

Reply:
Good catch!